nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -160,9 +163,13 @@ BtNavigator::loadBehaviorTree(const std::string & bt_xml_filename)
   RCLCPP_DEBUG(get_logger(), "Behavior Tree XML: %s", xml_string.c_str());
 
   // Create the Behavior Tree from the XML input
-  tree_ = bt_->buildTreeFromText(xml_string, blackboard_);
+  tree_ = bt_->createTreeFromText(xml_string, blackboard_);
```

Comment:
Change this name back please

Reply:
I thought it would make more sense to name it as the original function in BT.CPP V3.
https://github.com/BehaviorTree/BehaviorTree.CPP/blob/63d6fd42a1c4794c81aa7adfb5fba3e42aace8b8/src/bt_factory.cpp#L249

This is the only reference in the whole nav2-stack to the "old" buildTreeFromText function.

As this was the main reason why Groot failed until now (copying an existing (or later added) function from BT into nav2), I think it is worth a change. This way you can find the function faster in BT.CPP.

But if you say no, I could also live with that.
It was definitely intentional on my side.